articles/machine-learning/how-to-auto-train-image-models.md

```diff
@@ -359,7 +393,9 @@ In individual trials, you directly control the model architecture and hyperparam
 
 #### Supported model architectures
 
-The following table summarizes the supported models for each computer vision task.
+The following table summarizes the supported legacy models for each computer vision task. These models are compatible with the legacy runtime orchestration as well as the new pipelines orchestration.
+
+If you are using the pipelines orchestration, you can additionally use image classification models from the [HuggingFace Hub](https://huggingface.co/models?pipeline_tag=image-classification&library=transformers) which are part of the transformers library (such as microsoft/beit-base-patch16-224), as well as object detection and instance segmentation models from the [MMDetection Model Zoo](https://mmdetection.readthedocs.io/en/latest/model_zoo.html) (such as atss_r50_fpn_1x_coco). For curated models from HuggingFace and MMDetection, please visit the [Azure Machine Learning Model Catalog](concept-foundation-models.md).
```

Comment:
we should list curated models explictly and say what happens for those - like default hps, we made sure they work etc

Reply:
Please let me know what you think of the current table. If it looks good I'll start populating the OD and IS entries. We will need to update when the models are pushed to azureml staging